DigitalLearningSolutions.Web/Views/Register/Index.cshtml

```diff
@@ -15,25 +15,40 @@
 
       <div class="nhsuk-u-margin-bottom-8">
         <h1 id="page-heading" class="nhsuk-heading-xl">Register</h1>
+        <input type="hidden" asp-for="IsCentreSpecific" value=@Model.IsCentreSpecific/>
 
-        <p class="nhsuk-body-l">
-          Please select your centre to start the registration process
-          to access all Digital Learning Solutions services.
+        @if (Model.IsCentreSpecific)
+        {
+          <p class="nhsuk-body-l">
+            Please enter your personal details to start the registration process
+            to access all Digital Learning Solutions services at <b>@ViewBag.CentreName</b>.
+            <a class="nhsuk-action-link__link nhsuk-link--no-visited-state nhsuk-u-padding-left-0 nhsuk-body-m nhsuk-u-margin-top-2" asp-action="Index">
+              Not the right centre? Click here to change.
+            </a>
         </p>

```

Comment:
Minor, but the indentation seems to have gone wrong here. Should just be able to run the cleanup code to fix it.